DigitalLearningSolutions.Data/Services/LearningHubAuthApiUrlService.cs

```diff
@@ -0,0 +1,77 @@
+﻿namespace DigitalLearningSolutions.Data.Services

```

Comment:
There was a service added under HEEDLS-681, the `LearningHubLinkService`.
These two services could be comined.
Additionally that service contains a method that could be used to check if a user is already linked: `bool IsLearningHubAccountLinked(int delegateId);`